features/partners.feature

```diff
@@ -10,38 +10,121 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
-      Als de persoon met burgerservicenummer 999992806 wordt geraadpleegd met fields=partners
-      Dan bevat het antwoord GEEN partners
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992806 |
+      En de persoon heeft een partner met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992935 |
+      En de partner heeft de volgende naam gegevens
+      | naam              | waarde |
+      | voornamen (02.10) | Osama  |
+      En de partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
+      | naam                      | waarde   |
+      | datum aangaan (06.10)     | 20000115 |
+      | indicatie onjuist (84.10) |          |
+      En de partner heeft de volgende ontbindingHuwelijkPartnerschap gegevens
+      | naam                     | waarde   |
+      | datum ontbinding (07.10) | 20011109 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999993380                       |
+      | fields              | partners.naam.voornamen         |
+      Dan heeft de persoon met burgerservicenummer '999993380' GEEN 'partners'
 
     Scenario: Een actueel huwelijk en een ontbonden huwelijk
-      Gegeven de persoon met burgerservicenummer 999991553 heeft de volgende huwelijken/partnerschappen in de registratie
-        | Categorie | Voornamen | Datum aangaan (06.10) | Datum ontbinding (07.10) |
-        | 5         | Karel     | 20091102              |                          |
-        | 5         | Björn     |                       | 20080706                 |
-        | 55        | Björn     | 19870714              |                          |
-        | 55        | Björn     | 19870714              |                          |
-      Als de persoon met burgerservicenummer 999991553 wordt geraadpleegd met fields=partners
-      Dan bevat het antwoord 1 partner
-      En bevat het antwoord de partner met naam.voornamen "Karel"
-      En bevat het antwoord GEEN partner met naam.voornamen "Björn"
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999991553 |
+      En de persoon heeft een partner met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992935 |
+      En de partner heeft de volgende naam gegevens
+      | naam              | waarde |
+      | voornamen (02.10) | Karel  |
+      En de partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
+      | naam                      | waarde   |
+      | datum aangaan (06.10)     | 20091102 |
+      | indicatie onjuist (84.10) |          |
```

Comment:
en de partner heeft geen ontbindingHuwelijkPartnerschap gegevens

Reply:
Waarom moet dit worden opgenomen? Bij de opmerking van regel 71 is aangegeven bij een ontbinding (groep 7) dat er geen aangaan (groep6) is. Dan geldt het omgekeerde toch ook?

Reply:
daar heb je gelijk in, maar dat veronderstelt wel dat de lezer dit ook weet. Dus dat de lezer de consequentie begrijpt dat er dus geen ontbinding is.